go.mod

```diff
@@ -17,22 +18,22 @@ require (
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/ghodss/yaml v1.0.0
 	github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8 // indirect
-	github.com/go-kit/kit v0.8.0
+	github.com/go-kit/kit v0.0.0-20160928073314-fe6fe28ba0d5
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/go-openapi/analysis v0.17.2 // indirect
 	github.com/go-openapi/errors v0.17.2
-	github.com/go-openapi/jsonpointer v0.19.0 // indirect
-	github.com/go-openapi/jsonreference v0.19.0 // indirect
-	github.com/go-openapi/loads v0.17.2 // indirect
+	github.com/go-openapi/loads v0.17.2
 	github.com/go-openapi/runtime v0.17.2 // indirect
 	github.com/go-openapi/spec v0.17.2
 	github.com/go-openapi/strfmt v0.17.0
 	github.com/go-openapi/swag v0.19.0 // indirect
 	github.com/go-openapi/validate v0.18.0
-	github.com/gogo/protobuf v1.1.1
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
 	github.com/golang/mock v0.0.0-20190509174753-d74b93584564
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.2.0
```

Comment:
Probably not a big deal, but a few version numbers jumped backward. I assume this was on purpose...

Reply:
being honest that was output from `make deps-update` have no idea why it jumped backward, I will check that.